stack/setup/cloudflare.tf

```diff
@@ -33,3 +33,18 @@ resource "aws_acm_certificate_validation" "default" {
   certificate_arn         = aws_acm_certificate.main.arn
   validation_record_fqdns = cloudflare_record.validation.*.hostname
 }
+
+# If you came across this error:
+# Error: invalid zone setting "cname_flattening" (value: ) found - cannot be set as it is read only
+# 1. terraform state rm module.<module-name>.cloudflare_zone_settings_override.tls
+# 2. terraform apply
+# https://github.com/cloudflare/terraform-provider-cloudflare/issues/808
+resource "cloudflare_zone_settings_override" "tls" {
+  zone_id = data.cloudflare_zone.default.id
+
+  settings {
```

Comment:
should we additionally set `always_use_https` to "on"? (defaults to "off").

Always Use HTTPS
> Redirect all requests with scheme “http” to “https”. This applies to all http requests to the zone.

Reply:
Good idea! have added this to settings